bot/bot.py

```diff
@@ -0,0 +1,20 @@
+import os
```

Comment:
Please add [docstrings](https://www.python.org/dev/peps/pep-0257/) for this module and its functions. (This is causing the Travis build to fail at the moment.)